config/kubernetes/sig-release/teams.yaml

```diff
@@ -208,6 +208,10 @@ teams:
               the Kubernetes project. This team is a notification-only group,
               which includes the SIG Release Chairs for continuity.
             members:
+            - amwat
```

Comment:
Also maybe alpha? Unless we are taking over the whole block

Reply:
I think CI might fail on non-alpha anyway

Reply:
I think maybe the others should be removed but not clear.
humans alpha sorting things by hand is just the worst.